ext/fetch/lib.rs

```diff
@@ -266,15 +258,13 @@ where
         file_fetch_handler, ..
       } = state.borrow_mut::<Options>();
       let file_fetch_handler = file_fetch_handler.clone();
-      let (request, maybe_request_body, maybe_cancel_handle) =
+      let (request, maybe_cancel_handle) =
         file_fetch_handler.fetch_file(state, url);
       let request_rid = state.resource_table.add(FetchRequestResource(request));
-      let maybe_request_body_rid =
-        maybe_request_body.map(|r| state.resource_table.add(r));
       let maybe_cancel_handle_rid = maybe_cancel_handle
         .map(|ch| state.resource_table.add(FetchCancelHandle(ch)));
 
-      (request_rid, maybe_request_body_rid, maybe_cancel_handle_rid)
+      (request_rid, None, maybe_cancel_handle_rid)
```

Comment:
Do we need this `None` here?

Reply:
We do -- there's another path that creates the resource explicitly.